sources/api/pluto/src/main.rs

```diff
@@ -56,6 +58,11 @@ const DEFAULT_10_RANGE_DNS_CLUSTER_IP: &str = "172.20.0.10";
 
 const ENI_MAX_PODS_PATH: &str = "/usr/share/eks/eni-max-pods";
 
+/// The path to the AWS config file. The contents of settings.aws.config are decoded and written here.
+const AWS_CONFIG_FILE_PATH: &str = "/root/.aws/config.pluto";
```

Comment:
If this is temporary, you could store it at `/run` or `/tmp`

Reply:
Sure. I'll update this to use `tempfile` to create a `config.pluto` in a tempdir